schemas/questionnaire_flow/linear.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$id": "https://eq.ons.gov.uk/questionnaire_flow/linear.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "options": {
+    "type": "object",
+    "properties": {
+      "summary": {
+        "type": "object",
+        "description": "Using this property will include a summary on the submit page",
```

Comment:
The description doesn't really match the pattern currently used, they are more statements, rather than directions. "Summary on submit page" is probably fine. Arguably you might not even need a description